### ISSUE TITLE
Upgrade to RAP 3.28 / Java 17

### ISSUE THREAD

**Issue by ReneKellerAI** [eclipsesource/tabris#484]
Would it be possible to upgrade to the current Version of RAP (3.28) with Java 17 support? We're looking forward to updating our stack.

**Comment by ifurnadjiev** [comment]
Yes... There are no breaking changes in the last RAP release. I don't expect any problems to upgrade RAP framework by keeping the tabris bundle(s) as it is.
